SimpleWAWS/Code/CsmExtensions/CsmSiteExtensions.cs

```diff
@@ -27,16 +27,22 @@ public static partial class CsmManager
 
             site.AppSettings = config.properties.ToDictionary(k => k.name, v => v.value);
 
-            var properties = site.AppSettings.Select(e => new { name = e.Key, value = e.Value });
-            response = await csmClient.HttpInvoke(HttpMethod.Put, ArmUriTemplates.PutSiteAppSettings.Bind(site), new { properties = properties });
+            var properties = site.AppSettings.Select(e => new {name = e.Key, value = e.Value});
+            response =
+                await
+                    csmClient.HttpInvoke(HttpMethod.Put, ArmUriTemplates.PutSiteAppSettings.Bind(site),
+                        new {properties = properties});
             await response.EnsureSuccessStatusCodeWithFullError();
 
             return site;
         }
 
         public static async Task<Site> UpdateAppSettings(this Site site)
         {
-            var csmResponse = await csmClient.HttpInvoke(HttpMethod.Put, ArmUriTemplates.PutSiteAppSettings.Bind(site), new { properties = site.AppSettings.Select(s => new { name = s.Key, value = s.Value }) });
+            var csmResponse =
+                await
+                    csmClient.HttpInvoke(HttpMethod.Put, ArmUriTemplates.PutSiteAppSettings.Bind(site),
+                        new {properties = site.AppSettings.Select(s => new {name = s.Key, value = s.Value})});
```

Comment:
don't make a change that only affects the layout of the code if it's not needed.